[2024-12-06T11:31:21.527+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-06T11:31:21.540+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: telemadrid_schedule_scraper_v2.scrape_telemadrid_schedule scheduled__2024-01-29T11:32:00+00:00 [queued]>
[2024-12-06T11:31:21.545+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: telemadrid_schedule_scraper_v2.scrape_telemadrid_schedule scheduled__2024-01-29T11:32:00+00:00 [queued]>
[2024-12-06T11:31:21.545+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 5
[2024-12-06T11:31:21.572+0000] {taskinstance.py:2889} INFO - Executing <Task(_PythonDecoratedOperator): scrape_telemadrid_schedule> on 2024-01-29 11:32:00+00:00
[2024-12-06T11:31:21.577+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=270) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-12-06T11:31:21.579+0000] {standard_task_runner.py:72} INFO - Started process 284 to run task
[2024-12-06T11:31:21.580+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'telemadrid_schedule_scraper_v2', 'scrape_telemadrid_schedule', 'scheduled__2024-01-29T11:32:00+00:00', '--job-id', '717', '--raw', '--subdir', 'DAGS_FOLDER/dag_telemadrid_v2.py', '--cfg-path', '/tmp/tmpuk0c9yn3']
[2024-12-06T11:31:21.581+0000] {standard_task_runner.py:105} INFO - Job 717: Subtask scrape_telemadrid_schedule
[2024-12-06T11:31:21.614+0000] {task_command.py:467} INFO - Running <TaskInstance: telemadrid_schedule_scraper_v2.scrape_telemadrid_schedule scheduled__2024-01-29T11:32:00+00:00 [running]> on host d88abb179e8d
[2024-12-06T11:31:21.668+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='telemadrid_schedule_scraper_v2' AIRFLOW_CTX_TASK_ID='scrape_telemadrid_schedule' AIRFLOW_CTX_EXECUTION_DATE='2024-01-29T11:32:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-29T11:32:00+00:00'
[2024-12-06T11:31:21.669+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-06T11:31:21.899+0000] {logging_mixin.py:190} INFO - Escrito el xmltv del canal "Telemadrid"
[2024-12-06T11:31:25.081+0000] {logging_mixin.py:190} INFO - Error: 422 {"message": "Invalid request.\n\n\"sha\" wasn't supplied.", "documentation_url": "https://docs.github.com/rest/repos/contents#create-or-update-file-contents", "status": "422"}
[2024-12-06T11:31:25.082+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-12-06T11:31:25.088+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-06T11:31:25.088+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=telemadrid_schedule_scraper_v2, task_id=scrape_telemadrid_schedule, run_id=scheduled__2024-01-29T11:32:00+00:00, execution_date=20240129T113200, start_date=20241206T113121, end_date=20241206T113125
[2024-12-06T11:31:25.161+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-12-06T11:31:25.177+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-12-06T11:31:25.179+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
